src/app/OperationalDeviceProxy.h

```diff
@@ -132,6 +132,8 @@ class DLL_EXPORT OperationalDeviceProxy : public DeviceProxy, SessionReleaseDele
      */
     CHIP_ERROR Disconnect() override;
 
+    void SetConnectedSession(SessionHandle handle);
```

Comment:
Needs documentation about assumptions and when this is a reasonable thing to do.

Reply:
Added docs

Reply:
This does not document the "when this is a reasonable thing to do" bit.  Is it ok to do if the DeviceProxy already has a session, for example?  What will happen to communications using the existing session?  etc.

Reply:
Added comments about usage and deprecation